### DISCUSSION TITLE
Logo

### DISCUSSION THREAD

**Discussion by Yakui420** [davmac314/dinit#346] [Ideas]
I was bored so I was thinking about designing a logo for this project.
A few examples:
![dinit1](https://github.com/davmac314/dinit/assets/97893737/61622dbc-3edb-4f5a-8198-cf177173041f)
![dinit2](https://github.com/davmac314/dinit/assets/97893737/edb6e6e4-9012-4062-85db-5dcd72853e03)
![dinit4](https://github.com/davmac314/dinit/assets/97893737/112b4d79-7d3c-4253-8531-d4da2f3442f2)
![dinit5](https://github.com/davmac314/dinit/assets/97893737/6abbe482-4848-4762-8b12-c3204c86e610)

I tried to keep it minimal and simplistic to fit the theme. Choose your poison.

**Comment by davmac314** [comment]
This is cool! I like the first one a lot. @mobin-2008 what do you think?

**Comment by mobin-2008** [comment]
> I was bored so I was thinking about designing a logo for this project. A few examples: ![dinit1](https://private-user-images.githubusercontent.com/97893737/340655436-61622dbc-3edb-4f5a-8198-cf177173041f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3MTc2MjQsIm5iZiI6MTcxODcxNzMyNCwicGF0aCI6Ii85Nzg5MzczNy8zNDA2NTU0MzYtNjE2MjJkYmMtM2VkYi00ZjVhLTgxOTgtY2YxNzcxNzMwNDFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE4VDEzMjg0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY4MThkOTI4ZTM1ZjVkYWNkM2E2ZTI3OGZhNTU1NTkxMDEyMzZjMjk0MTU2YzE0ODY3MWU1YjY3OGU0NWNkYTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Sp-3D_Yrm3M85nCJoEmcbp9FpeDNkO08A_XcqhWcSTI) ![dinit2](https://private-user-images.githubusercontent.com/97893737/340655244-edb6e6e4-9012-4062-85db-5dcd72853e03.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3MTc2MjQsIm5iZiI6MTcxODcxNzMyNCwicGF0aCI6Ii85Nzg5MzczNy8zNDA2NTUyNDQtZWRiNmU2ZTQtOTAxMi00MDYyLTg1ZGItNWRjZDcyODUzZTAzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE4VDEzMjg0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU3ZTRmNzRkNTQ2NDUxYjZmYjAzZWQ1ZmQ4NjAzY2E0OWFiYmMzYWM0M2NlOTAyZDE1YmQ3NzhhYWNjMTA3MTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.51mWdIvI3olIrMAkeNCC9vzoEv1-85h_8OTYZC7E2Zk) ![dinit4](https://private-user-images.githubusercontent.com/97893737/340655245-112b4d79-7d3c-4253-8531-d4da2f3442f2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3MTc2MjQsIm5iZiI6MTcxODcxNzMyNCwicGF0aCI6Ii85Nzg5MzczNy8zNDA2NTUyNDUtMTEyYjRkNzktN2QzYy00MjUzLTg1MzEtZDRkYTJmMzQ0MmYyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE4VDEzMjg0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRlNDk2MGU4YzAzNDk5ZGJlY2YyNGZiZTdhMTM2NTFhNzMxMzNhMDc1NTg5ZmJlYmUyYmVmOTRmMGU3MWIxY2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.SnRl6adS4uYyZNX8R0LYc4O9NXWhEUtuvd1bkB16j_U) ![dinit5](https://private-user-images.githubusercontent.com/97893737/340655248-6abbe482-4848-4762-8b12-c3204c86e610.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3MTc2MjQsIm5iZiI6MTcxODcxNzMyNCwicGF0aCI6Ii85Nzg5MzczNy8zNDA2NTUyNDgtNmFiYmU0ODItNDg0OC00NzYyLThiMTItYzMyMDRjODZlNjEwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE4VDEzMjg0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc1MmJhYjkzNmVhZjk0OGVlNzE0ODRmYTM5YjUwYzMzNGNmYWNjNWQ0MmY5ZmFlMTgwNDZhYWIzMTQ3MDg2ODkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.R9UtGQShXqJmPAswjaH3xIxPsuvQAjbHdMAiqiU_ZrM)
> 
> I tried to keep it minimal and simplistic to fit the theme. Choose your poison.

Thanks, Both look cool but I really enjoyed first one.

**Comment by davmac314** [comment]
@Yakui420 just to be clear, are you ok with us using the logo? What license is it under?

**Reply by Yakui420** [reply]
Sure, that's one of the reasons why I made it. Everyone is free to use it without crediting.

**Reply by davmac314** [reply]
Great, thanks! How should we credit you - as **Yakui420** or something else?

**Reply by davmac314** [reply]
(I know you said credit isn't required. But I would definitely like to acknowledge the contribution. If you would prefer not, that is fine too, just let me know).

**Reply by Yakui420** [reply]
Yeah, Yakui420.

**Comment by Yakui420** [comment]
Oh and as expected, the logo doesn't look well when dark mode is enabled.
I added a white background to fix this issue:
![dinit](https://github.com/user-attachments/assets/9adb0fc0-842b-471b-ba3e-70604070e2cf)

**Comment by Suzuco** [comment]
What about an extra outline so that it stands out in (probably) every situation? Just wondering, no offence of any kind intended.
![dinit-outline](https://github.com/user-attachments/assets/1b6b3bd8-e1bc-4494-8db4-cc8b0ef5de04)

**Reply by Yakui420** [reply]
Thanks.

**Reply by davmac314** [reply]
Thanks, I've updated the logo in the repo now.

**Reply by softexpert** [reply]
I had a few spare minutes and I come up with these SVG attempts to reproduce the chosen logo.
I don't know if they have any value, if you feel that they should be dropped or rectified, please let me know ...

![dinit_logo_artix](https://github.com/user-attachments/assets/bb423e04-deb3-408d-8238-5fcf03df7504)
![dinit_logo_artix2](https://github.com/user-attachments/assets/460bf85a-dabf-4af7-9c71-35dc2318e086)
![dinit_logo_black](https://github.com/user-attachments/assets/315233c6-175b-4ad1-bbf9-f923a16d639d)
![dinit_logo_chimera](https://github.com/user-attachments/assets/4cabd8c1-b308-47cb-b7b1-ce9070399f45)
![dinit_logo_gray](https://github.com/user-attachments/assets/f58f8672-ba4b-444f-9b1b-76bc3bccdd24)

**Reply by Yakui420** [reply]
> What about an extra outline so that it stands out in (probably) every situation? Just wondering, no offence of any kind intended. ![dinit-outline](https://private-user-images.githubusercontent.com/7494910/350512849-1b6b3bd8-e1bc-4494-8db4-cc8b0ef5de04.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAyOTA2ODMsIm5iZiI6MTczMDI5MDM4MywicGF0aCI6Ii83NDk0OTEwLzM1MDUxMjg0OS0xYjZiM2JkOC1lMWJjLTQ0OTQtOGRiNC1jYzhiMGVmNWRlMDQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMzBUMTIxMzAzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODE2YzhlNjk1YWMyNzljN2RkZGEyZWViZmFmNjE5NzRkNDI0ODU0YzE5YWFjZjdiNTRhOTNiZjZjZTgxZmI2OCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.OMFFI2Akk_NV03pSJqgDx8Qe59FkcBf4_ZjvGoLUWD8)

Oh and to answer your question, I was on the phone at the time so I couldn't edit an extra outline.

**Reply by Yakui420** [reply]
> I had a few spare minutes and I come up with these SVG attempts to reproduce the chosen logo. I don't know if they have any value, if you feel that they should be dropped or rectified, please let me know ...

This looks very nice. I really like the first blue one.
